packages/plugins/experimental/plugin-kanban/src/components/KanbanContainer.tsx

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React, { useEffect, useState } from 'react';
+
+import { type MutableSchema } from '@dxos/echo-schema';
+import { useGlobalFilteredObjects } from '@dxos/plugin-search';
+import { Filter, useQuery, getSpace } from '@dxos/react-client/echo';
+import { type KanbanType, useKanbanModel, Kanban } from '@dxos/react-ui-kanban';
+import { StackItem } from '@dxos/react-ui-stack';
+
+import { stateColumns } from './create-kanban';
+
+export const KanbanContainer = ({ kanban, role }: { kanban: KanbanType; role: string }) => {
```

Comment:
i think it's worth export KanbanContainerProps

Reply:
Most of the main plugins are already using `StackItem.Content` instead of other grid logic. Container is also a good opportunity to load models and display a fallback meanwhile if needed.